src/components/atoms/IconInput/IconInput.scss

```diff
@@ -0,0 +1,41 @@
+@import "scss/constants";
+
+.icon-input {
+  position: relative;
+
+  &__input {
+    height: 100%;
+
+    &--icon-from-left {
```

Comment:
```suggestion
    &--icon-left {
```

Reply:
It might be better to move these variants onto the base class itself, so that they are:

- `.icon-input--icon-left`
- `.icon-input--icon-right`

or similar.